adapters/videobyte/videobyte_test.go

```diff
@@ -0,0 +1,11 @@
+package videobyte
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+)
+
+func TestJsonSamples(t *testing.T) {
+	adapterstest.RunJSONBidderTest(t, "videobytetest", &VideoByteAdapter{endpoint: "https://mock.videobyte.com"})
```

Comment:
Please consider using the `Builder` function to get an instance of the adapter to be passed into the `RunJSONBidderTest` function so that there's test coverage for that function too.

Reply:
Done.